src/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -23,8 +23,10 @@ internal class MSBuildForwardingAppWithoutLogging
             new Dictionary<string, string>
             {
                 { "MSBuildExtensionsPath", AppContext.BaseDirectory },
```

Comment:
These are environment variables passed down to the msbuild process created by the CLI whenever it shells out to msbuild.